test/teos/unit/cli/test_rpc_client.py

```diff
@@ -0,0 +1,17 @@
+import pytest
```

Comment:
I know I told you to test the rpc_client, but is there any point on testing it like this? Just wondering.

Reply:
Well, the `get_user` function has some logic beyond just proxying to the rpc service (raising InvalidParameter), so I think it makes sense to have a unit test for it, I think.

Reply:
Oh OK, fair enough.